src/main/resources/org/joda/money/MoneyData.csv

```diff
@@ -23,6 +23,7 @@ BRL,986,2,BR
 BSD,44,2,BS
 BTN,64,2,BT
 BWP,72,2,BW
+BYN,-1,2,BY
```

Comment:
Now I see that there may not be multiple currency definitions for same country. Any ideas what to do if we still need to construct Money object with both currencies?